test/integration/apiserver/apiserver_test.go

```diff
@@ -3352,3 +3353,23 @@ func assertManagedFields(t *testing.T, obj *unstructured.Unstructured) {
 func int32Ptr(i int32) *int32 {
 	return &i
 }
+
+func parseFeatureGates(metricsBody string) (map[string]int, error) {
```

Comment:
nit: helper should probably also be in the same file as the rest of the tests?

Reply:
Good catch - this was not intended.  Fixed now